src/main/java/uk/gov/hmcts/darts/arm/component/ArchiveRecordFileGenerator.java

```diff
@@ -9,5 +9,5 @@
 public interface ArchiveRecordFileGenerator {
     boolean generateArchiveRecord(ArchiveRecord archiveRecord, File archiveRecordFile, ArchiveRecordType archiveRecordType);
 
-    void generateArchiveRecords(List<ArchiveRecord> archiveRecords, File archiveRecordsFile);
+    String generateArchiveRecords(String archvieFileName, List<ArchiveRecord> archiveRecords);
```

Comment:
Typo in archive. This appears in various lines.

Reply:
Good spot updated :)